markdown/publications.md

```diff
@@ -69,13 +69,13 @@ Article in _Nature_ describing scientific workflows (including nf-core):
 
 ### [nf-core/eager](https://nf-co.re/eager)
 
-<!-- pub-stats 10.1101/2020.06.11.145615v1 -->
+<!-- pub-stats 10.7717/peerj.10947 -->
 > **Reproducible, portable, and efficient ancient genome reconstruction with nf-core/eager**
 >
-> James A. Fellow Yates, Thiseas Christos Lamnidis, Maxime Borry, Aida Andrades Valtueña, Zandra Fagernäs, Stephen Clayton, Maxime U. Garcia, Judith Neukamm, Alexander Peltzer
+> James A. Fellow Yates, Thiseas C. Lamnidis, Maxime Borry, Aida Andrades Valtueña, Zandra Fagernäs, Stephen Clayton, Maxime U. Garcia, Judith Neukamm, Alexander Peltzer
```

Comment:
Maybe?

```suggestion
> James A. Fellow Yates, Thiseas C. Lamnidis, Maxime Borry, Aida A. Valtueña, Zandra Fagernäs, Stephen Clayton, Maxime U. Garcia, Judith Neukamm, Alexander Peltzer
```

Reply:
Nope, Spanish == two surnames ;) although just saw a typo in my own name :facepalm: